src/formattedcode/format_spdx_tv.py

```diff
@@ -0,0 +1,188 @@
+#
```

Comment:
Instead of two hook implementations, how about defining the same plugin for both the formats in the `scancode_formats` entrypoint? Then the plugin can differentiate between the formats using the `format` parameter you referred to in a previous comment.